app/helpers/otus_helper.rb

```diff
@@ -23,14 +23,51 @@ def otu_tag_elements(otu)
   end
 
   # Used exclusively in /api/v1/otus/autocomplete
-  def otu_extended_autocomplete_tag(target)
+  def otu_extended_autocomplete_tag(target, otu, term, include_common_names)
```

Comment:
@kleintom I'd like to refactor this.  the last param should be an array of common names to include.  This lets us find all common names for a set of OTUs *before* we get here in a single query, removing a lot of n+1 queries.